app/assets/stylesheets/pin-show.css.scss

```diff
@@ -0,0 +1,117 @@
+$primary-background-color: rgba(237, 242, 242, 0.6);
+
+
+.container-fluid{
```

Comment:
Yep but you might need another container somewhere else for some reason. General rule is: don't change their rules, but add others on top, by adding a new class: then do ``` .container-fluid.my-new-class {    } ```